lib/motions/general-motions.coffee

```diff
@@ -289,6 +289,36 @@ class MoveToEndOfWord extends Motion
 class MoveToEndOfWholeWord extends MoveToEndOfWord
   wordRegex: WholeWordRegex
 
+class MoveToNextSentence extends Motion
+  moveCursor: (cursor, count=1) ->
+    _.times count, ->
+      start = cursor.getBufferPosition().translate new Point(0, 1)
+      eof = cursor.editor.getBuffer().getEndPosition()
+      scanRange = [start, eof]
+
+      cursor.editor.scanInBufferRange /(^$)|(([\.!?] )|^[A-Za-z0-9])/, scanRange, ({matchText, range, stop}) ->
```

Comment:
Update on testing. Could not correctly implement the behavior when removing the letters from the regex. I'd suggest it could go as an improvement for a later PR.